common-v2/all-prefix.js

```diff
@@ -2,13 +2,27 @@
 // Prevent jQueryBE to be register in AMD
 // which cause conflict when client site use RequireJS
 if (typeof define === "function" && define.amd) {
-    window.tmpDefine = define; 
+    window.tmpDefine = define;
     define = undefined;
 }
 
 (function () {
-    if (document.all && !document.addEventListener) {
```

Comment:
To check if the browser version is below IE8, this condition is incorrect so it should be replaced by a new condition.

Reply:
This is one of the causes of BE-2554.